Hyokune/bonfire
# #4 Add electron-builder to bundle application

Closes: #4

How to test this PR:

1. Checkout branch locally
2. Check that running `npm run package` bundles the application
3. Check that the bundled executable runs correctly
4. Profit
